contribs/gmf/src/directives/search.js

```diff
@@ -205,6 +208,17 @@ gmf.SearchController = function($scope, $compile, $timeout, gettextCatalog,
   this.timeout_ = $timeout;
 
   /**
+   * @type {ngeo.Location}
+   * @private
+   */
+  this.ngeoLocation_ = $injector.get('ngeoLocation');
```

Comment:
I think we should not put a strong dependency between the search and the ngeoLocation, the search should work even if no ngeoLocation is present.

Please change to something like:
```js

/**
 * @type {ngeo.Location|undefined}
 * @private
 */
this.ngeoLocation_;

if ( $injector.has('ngeoLocation')) {
  this.ngeoLocation_ = $injector.get('ngeoLocation');
}
```